src/libraries/System.Drawing.Primitives/tests/ColorTests.cs

```diff
@@ -889,5 +889,27 @@ private static int GetColorRefValue(Color color)
             // The COLORREF value has the following hexadecimal form: 0x00bbggrr.
             return color.B << 16 | color.G << 8 | color.R;
         }
+
+        [Fact]
+        public void SystemColor_AlternativeColors()
+        {
+            try
+            {
+#pragma warning disable SYSLIB5002 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+                Drawing.SystemColors.UseAlternativeColorSet = true;
+#pragma warning restore SYSLIB5002
+
+                Assert.Equal(0xFF464646, (uint)Drawing.SystemColors.ActiveBorder.ToArgb());
+                Assert.Equal(0xFFF0F0F0, (uint)Drawing.SystemColors.WindowText.ToArgb());
+                Assert.Equal(0xFF202020, (uint)Drawing.SystemColors.ButtonFace.ToArgb());
+                Assert.Equal(0xFF2A80D2, (uint)Drawing.SystemColors.MenuHighlight.ToArgb());
+            }
+            finally
+            {
+#pragma warning disable SYSLIB5002 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+                Drawing.SystemColors.UseAlternativeColorSet = false;
+#pragma warning restore SYSLIB5002
+            }
+        }
```

Comment:
Any other tests that might be interesting?  I see that `ColorTranslator.FromOle` will eventually call into `KnownColorTable`

Reply:
Only "web" colors are impacted on that code path. The only other test would be to force the system into high contrast, but that is really fragile, and we get regular indirect coverage with accessibility test passes.